.gitignore

```diff
@@ -19,12 +20,10 @@ coverage.xml
 sqlnet.log
 /mapping_setup.py
 /test.py
+/test?.py
 /.cache/
 /.mypy_cache
 *.sw[o,p]
-/test?.py
-/test.py
-/mapping_setup.py
```

Comment:
I use this one...

Reply:
`/test.py` and `/mapping_setup.py` were flagged by PyCharm as dupes of what are now lines 21 and 22, so removed the second instances. I moved `/test?.py` up to be below `/test.py`.

Reply:
ah it's duped OK